model/browser/events.yaml

```
@@ -0,0 +1,83 @@
+groups:
+  - id: event.enduser.user_action
+    stability: development
+    type: event
+    name: enduser.user_action
+    brief: >
+      This event describes actions performed by the user such as click, scroll, zoom, resize, etc.
+    body:
+      id: enduser.user_action
+      requirement_level: required
+      stability: development
+      type: map
+      fields:
+        - id: element
+          type: string
+          stability: development
+          brief: Target element tag name and it is obtained via `event.target.tagName`.
+          requirement_level: recommended
+          examples: ["button"]
+        - id: xpath
+          type: string
+          stability: development
+          brief: Target element xpath
+          requirement_level: recommended
+          examples: ["//*[@id='testBtn']"]
+        - id: user_action_type
+          type: enum
+          members:
+            - id: clickleft
+              value: "clickleft"
```

(see convention we're using in db.system.name now)

```suggestion
            - id: click.left
              value: "click.left"
```

---

changed
`clickright` to `click.right`
`clickleft` to `click.left`
`clickmiddle` to `click.middle`